src/ros/build-env-utils.ts

```diff
@@ -68,23 +68,27 @@ async function updateCppPropertiesInternal(): Promise<void> {
         });
     }));
 
-    let platform: string = os.platform();
-    let c_cpp_properties: any = {
+    if (process.platform === "win32") {
+        includes.push(path.join("C:", "opt", "rosdeps", "x64", "include", "**"));
```

Comment:
this is necessary for dependencies installed by `rosdeps`

do you think similar change would also be needed for `vcpkg`?

Reply:
It is nice to have them to be in the include path, but I am not sure what's the best way to include them. Maybe we could enumerate all the `CMAKE_PREFIX_PATH` (instead of the hardcode path)?